src/rollingBall.ts

```diff
@@ -32,7 +40,7 @@ export function rollingBall(spectrum, options = {}) {
     throw new TypeError('Spectrum must not be empty');
   }
 
-  const numberPoints = spectrum.length;
+  const numberPoints: number = spectrum.length;
```

Comment:
You don't need to add a type annotation here. TS knows that spectrum is an array, so it knows that it has a property `length` which is a number